pkg/util/github/github.go

```diff
@@ -311,29 +313,20 @@ func downloadHash(release *github.RepositoryRelease, filename string) (string, e
 		return "", errors.Wrap(err, "Cannot download the OpenShift release checksum asset.")
 	}
 	if len(url) > 0 {
-		glog.V(2).Infof("Downloading OpenShift %s checksums\n", *release.TagName)
+		fmt.Printf("-- Downloading OpenShift %s checksums ... ", *release.TagName)
 		httpResp, err := http.Get(url)
 		if err != nil {
 			return "", errors.Wrap(err, "Cannot download the OpenShift release checksum asset.")
 		}
 		defer func() { _ = httpResp.Body.Close() }()
 
 		asset = httpResp.Body
-		if httpResp.ContentLength > 0 {
-			bar := pb.New64(httpResp.ContentLength).SetUnits(pb.U_BYTES)
-			bar.Start()
-			asset = bar.NewProxyReader(asset)
-			defer func() {
-				<-time.After(bar.RefreshRate)
-				fmt.Println()
-			}()
-		}
 	}
-
 	scanner := bufio.NewScanner(asset)
 	for scanner.Scan() {
 		spl := strings.Fields(scanner.Text())
-		if len(spl) == 2 && spl[1] == filename {
+		if len(spl) == 2 && strings.Contains(spl[1], filename) {
```

Comment:
@praveenkumar Why  specific reason `spl[1] == filename` does not work here?

Reply:
@LalatenduMohanty because you never predict what would be exact filename from the release of origin side check the older and newer releases and you find it out.

Reply:
@praveen makes sense.